### PR TITLE
AP-16 | [BE] Add Health check API

### PR DESCRIPTION
**Pull request by rdhawladar**

### CODE REVIEW

**Comment by zrshishir** [comment]
internal/app/model/entities.go this file should contain different model. Do we need internal/app/model/jwt.go file here in this service? Except these two files, it looks good to me.

File: internal/app/server/server.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	log.Info().Msg("shutting down server")
+	if err := s.http.Shutdown(context.Background()); err != nil {
+		if err != http.ErrServerClosed {
+			return err
+		}
+	}
+	return nil
+}
+
+func (s *server) pingHandler(w http.ResponseWriter, r *http.Request) {
```
Comment (shuza) [inline internal/app/server/server.go:54]: write test case for `pingHandler`

**Review by shuza — CHANGES_REQUESTED** [review]
add test for `pingHandler`

File: internal/app/server/server_test.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+		r, _ := http.NewRequest(http.MethodGet, "/", nil)
+
+		s.ServeHTTP(w, r)
+
+		assert.Equal(t, w.Body.String(), "404 page not found\n")
+		assert.Equal(t, w.Code, http.StatusNotFound)
+	})
+}
+
+func TestShutdown(t *testing.T) {
```
Comment (shuza) [inline internal/app/server/server_test.go]: why this test is empty?

**Review by shuza — COMMENTED** [review]
test function should test something otherwise remove empty test function

File: internal/app/model/entities.go [hunk 3]
```diff
@@ -1,4 +1 @@
 package model
```
Comment (Asad-noor) [inline internal/app/model/entities.go:1]: If there is no code then should we keep the file?

Comment (Asad-noor) [reply internal/app/model/entities.go:1]: Also, there are some other files and folders which are empty, if are not needed yet, we should remove them.

**Pull request merged**